Lesson3.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace Lesson3
+{
+    public class Lesson3
+    {
+	    public Lesson3()
+	    {
+            static void Main(string[] args)
+            {
+                Get_Attributes();
+                Print_Description();
+            }
+            static void Get_Attributes()
+            {
+                string name = PromptString("name");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'name'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971681)